docs/Plugins.rst

```diff
@@ -2317,7 +2320,13 @@ all valid plant IDs will be listed.
     This leads to some shrubs being designated when they shouldn't be, causing a
     plant gatherer to walk there and do nothing (except clearing the
     designation). See :issue:`1479` for details.
-
+    
+    The implementation another known deficiency: it's incapable of detecting that
+    RAW definitions that specify a seed extraction reaction for the structural part
+    but has no other use for it cannot actually yield any seeds, as the part is
+    never used (parts of bug tracker report 6940, e.g. Red Spinach), even though DF
```

Comment:
For reference, similar to `:issue:` in the paragraph above (which links to DFHack issues on GitHub), `:bug:` links to Mantis issues. A full list can be found in [conf.py](https://github.com/DFHack/dfhack/blob/1546d9ec1a3c5673dfa49795bb8a956bf17c3b2d/conf.py#L199-L209)